modules/dns/versions.tf

```diff
@@ -6,6 +6,9 @@ terraform {
     linode = {
       source = "linode/linode"
     }
+    aws = {
+      source = "hashicorp/aws"
+    }
   }
   required_version = ">= 0.13"
```

Comment:
The required_version has been bumped to 1.7 at the root module. The DNS submodule's lower requirements are likely accurate but could also be bumped.